dom.js

```diff
@@ -418,7 +418,7 @@
 
     // HTML - https://html.spec.whatwg.org
     // Element.classList
-    if ('classList' in document.createElement('span')) {
+    if ('classList' in document.createElementNS('http://www.w3.org/2000/svg', 'svg')) {
```

Comment:
This line is trying to answer: "when installing a helper that can be used even in IE7-, is the classList property available on elements, or does a DOMTokenListShim need to be created explicitly?"

This line would only need to be changed if there was a version of IE7- that supported the classList property on span but not on svg, which is not the case.

Reply:
I'm not sure I understand. `.classList` doesn't work on SVG elements in higher versions of IE than 7, so wouldn't you'd want to make the check on an SVG element?

Reply:
In IE8 and higher the `classList` getter polyfill can be added to `Element prototype` (see later code) so the `window.getClassList` helper can just call `elem.classList`